overlord/snapstate/handlers_link_test.go

```diff
@@ -1504,6 +1504,56 @@ func (s *linkSnapSuite) TestDoUndoLinkSnapRestoresRefreshInhibitedTime(c *C) {
 	c.Check(snapst.RefreshInhibitedTime.Equal(instant), Equals, true)
 }
 
+func (s *linkSnapSuite) TestDoUndoLinkSnapRestoresLastRefreshTime(c *C) {
```

Comment:
I think we are missing the equivalent of TestLinkSnapResetsRefreshInhibitedTime about just the do part, basically something like TestLinkSnapSetsLastRefreshTime

Reply:
Sure, I'll add a more explicit test.

Reply:
Done